README.md

```diff
@@ -1,34 +1,52 @@
-# Gunicorn Operator
+# Gunicorn-k8s Operator
 
-## Description
+A Juju charm deploying docker images using gunicorn as default. 
 
-A charm that allows you to deploy your gunicorn application in kubernetes.
+This charm simplifies docker image deployment management, allowing us to inject variables to the environment as we see fit. It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
+offerings.
 
-## Usage
+As such, the charm makes it easy for those looking to take control of their docker images, use them in a juju environment without having to write a charm from scratch and gives them the
+freedom to deploy on the Kubernetes platform of their choice.
 
-By default, the charm will deploy a simple [OCI](https://opencontainers.org/)
-image that contains a gunicorn app that displays a short message and its
-environment variables. The image is built using an OCI Recipe on Launchpad and
-published to dockerhub [here](https://hub.docker.com/r/gunicorncharmers/gunicorn-app).
-```
-juju deploy gunicorn-k8s my-awesome-app
-```
+For DevOps or SRE teams this charm will make docker image testing easier and more manageable. It will allow easy deployment
+into multiple environments for testing of changes, and supports scaling out for
+enterprise deployments.
 
-### Scale Out Usage
+## Deployment options overview
 
-```
-juju add-unit my-awesome-app
-```
+For overall concepts related to using Juju
+[see the Juju overview page](https://juju.is/). For easy local testing we
+recommend
+[this how to on using MicroK8s with Juju](https://juju.is/docs/microk8s-cloud).
 
-## OCI image
+## How to deploy this charm (quick guide)
 
-### Using your own image
+To deploy the charm and relate it to
+[the PostgreSQL K8s charm](https://charmhub.io/postgresql-k8s) within a Juju Kubernetes model:
```

Comment:
Should we mention MongoDB too?

Reply:
Not just yet, we will wait until the mongodb cham is in a more mature state.